canonical/canonical.com
# Build faq page

## Done

Build an FAQ page on `/careers/application/faq`
Demo at https://canonical-com-619.demos.haus/careers/application/faq

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare page against [copy doc](https://docs.google.com/document/d/12_O_QtPqfv9p7kaDFdiP8Xq-kY5xa1XB-C-CRbXjDew/edit) and [design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?node-id=5916%3A8133)

## Issue / Card

Fixes #617 

## Screenshots

<img width="1440" alt="Publisher of Ubuntu | Canonical ()" src="https://user-images.githubusercontent.com/57550290/187693866-0406182d-a0fc-4440-85bf-f679ca7db126.png">
